### PR TITLE
Provide proper certificate path to admin console

### PR DESCRIPTION
**Pull request by kwoodson**
Fix for the following issue:
https://github.com/Azure/OpenShift/issues/134

```release-note:
Fix for admin console certificate redirect
```

### CODE REVIEW

**Comment by kwoodson** [comment]
@ehashman  Did you want a back port for this one or do you want to push it forward for v16 only?  This was the other issue Anubhuti cited that she wanted fixed.  WDYT?

**Comment by olga-mir** [comment]
flake: https://github.com/openshift/openshift-azure/issues/2234
I am trying to figure out what happens. there seems to be a problem with linking the cluster and it fails probably all the time now

**Comment by kwoodson** [comment]
Upgrade 10.1.1 failed on a storage account createorupdate.
```
time="2020-02-19T18:58:48Z" level=fatal msg="containerservice.OpenShiftManagedClustersClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: error response cannot be parsed: \"400 Bad Request: Failed to apply request: CreateOrUpdateConfigStorageAccount: Put https://sacfg4v7y557ncu5mh5ruxq4.blob.core.windows.net/config?restype=container: dial tcp: lookup sacfg4v7y557ncu5mh5ruxq4.blob.core.windows.net on 10.142.0.80:53: no such host\\n\" error: json: cannot unmarshal number into Go value of type azure.RequestError"
```

**Comment by ehashman** [comment]
> @ehashman Did you want a back port for this one or do you want to push it forward for v16 only? This was the other issue Anubhuti cited that she wanted fixed. WDYT?

No backport, this should go into v16.

File: pkg/cluster/hash_test.go [hunk 1]
```diff
@@ -249,7 +249,7 @@ func TestHashSyncPodStability(t *testing.T) {
 		},
 		"v15.0": {
 			// this value should not change
-			expectedHash: "5d4affe409e41f7fe5cf52af7968f5c96c911a9dfe34aad07b6d9575615ec2a8",
+			expectedHash: "6bdb841bbe9c18d4d7005fb5a5251428a39ce8a328057ea41a6762b55590f97e",
```
Comment (ehashman) [inline pkg/cluster/hash_test.go]: Please revert changes on v15 so the hashes don't change.

Comment (kwoodson) [reply pkg/cluster/hash_test.go]: Removed v15 updates.

**Comment by kwoodson** [comment]
@ehashman Deploying a private link cluster, logging into the console, then verifying that when selecting the admin console the redirect issue has been resolved.

**Comment by kwoodson** [comment]
I've seen this flake a few times.  This appears to be a timeout when attempting to deploy images.
```
              Bucket: "checkCanDeployRedhatIoImages",
             Err: {
                  ErrStatus: {
                      TypeMeta: {Kind: "", APIVersion: ""},
                      ListMeta: {SelfLink: "", ResourceVersion: "", Continue: ""},
                      Status: "Failure",
                      Message: "Timeout: request did not complete within allowed duration",
                      Reason: "Timeout",
                      Details: {Name: "", Group: "", Kind: "", UID: "", Causes: nil, RetryAfterSeconds: 0},
                      Code: 504,
                  },
```

**Comment by kwoodson** [comment]
```
Ran 29 of 40 Specs in 2591.678 seconds
SUCCESS! -- 29 Passed | 0 Failed | 0 Pending | 11 Skipped
--- PASS: TestE2E (2591.69s)
PASS
ok  	github.com/openshift/openshift-azure/test/e2e	2600.569s
```

**Comment by kwoodson** [comment]
Looks like a timeout when reimaging master-000001.  The good news is that the first master came back online.

```
aiting for master-000001 to be ready" func="pkg/cluster.(*Upgrade).UpdateMasterAgentPool()" file="pkg/cluster/update_master.go:70"
time="2020-02-22T00:00:25Z" level=warning msg="net/http: TLS handshake timeout: retry 1" func="pkg/util/roundtrippers.(*RetryingRoundTripper).RoundTrip()" file="pkg/util/roundtrippers/retrying.go:141"
time="2020-02-22T00:01:21Z" level=warning msg="&errors.errorString{s:\"unexpected status code 400\"}: not retrying" func="pkg/util/roundtrippers.(*RetryingRoundTripper).RoundTrip()" file="pkg/util/roundtrippers/retrying.go:151"
time="2020-02-22T00:01:22Z" level=debug msg="400 Bad Request: Failed to apply request: UpdateMasterAgentPoolWaitForReady: Get https://10.0.0.254/api/v1/nodes/master-000001: unexpected status code 400" func="pkg/fakerp.(*Server).badRequest()" file="pkg/fakerp/util.go:24"
```

/test upgrade-private-v14.1

**Comment by kwoodson** [comment]
Failed due to a timeout.
```
  /home/prow/go/src/github.com/openshift/openshift-azure/test/e2e/specs/end_user.go:28
  Expected
      <[]*standard.TestError | len:1, cap:1>: [
          {
              Bucket: "checkCanUseAzureFile",
              Err: {
                  ErrStatus: {
                      TypeMeta: {Kind: "", APIVersion: ""},
                      ListMeta: {SelfLink: "", ResourceVersion: "", Continue: ""},
                      Status: "Failure",
                      Message: "Timeout: request did not complete within allowed duration",
                      Reason: "Timeout",
                      Details: {Name: "", Group: "", Kind: "", UID: "", Causes: nil, RetryAfterSeconds: 0},
                      Code: 504,
                  },
              },
          },
      ]
  to be empty
```

/test  upgrade-private-v14.1

**Pull request merged**